static/js/public/vueComponents/browseDisplay.vue

```diff
@@ -0,0 +1,260 @@
+        </pagination>
+    </div>
+</template>
+
+<script>
+    const COMPONENT_PATH = 'vue!public/vueComponents';
+    const MODULES = [
+        'Vue',
+        `${COMPONENT_PATH}/modalMetadata`,
+        `${COMPONENT_PATH}/browseSearch`,
+        `${COMPONENT_PATH}/browseSort`,
+        `${COMPONENT_PATH}/pagination`
+    ];
+
+    define(MODULES, function(Vue) {
+        Vue.component('browseDisplay', {
+            props: {
+                path: String,
+
+                recordsPerPage: {
+                    type: Number,
+                    default: 20
+                },
+
+                type: String
+            },
+
+            template: template,
+
+            data: function() {
+                return {
+                    column_size: 'is-3',
+                    container_metadata: {},
+                    pagination_settings: {},
+                    record_list: []
+                }
+            },
+
+            computed: {
+                childTypeText: function() {
+                    if (this.type === 'AdminUnit') {
+                        return 'collections';
+                    } else if (this.type === 'Collection') {
+                        return 'folders and files';
+                    } else {
+                        return 'files';
+                    }
+                },
+
+                typeText: function() {
+                    if (this.type === 'AdminUnit') {
+                        return 'administrative unit';
+                    }
+
+                    return this.type.toLowerCase();
+                },
+
+                displayList: function() {
+                    return this.record_list.slice(
+                        this.pagination_settings.start,
+                        this.pagination_settings.end
+                    );
+                },
+
+                displayNumber: function() {
+                    let page_display = this.pagination_settings.start + this.recordsPerPage;
+
+                    if (this.numberOfRecords < page_display) {
+                        return this.numberOfRecords;
+                    }
+
+                    return page_display;
+                },
+
+                numberOfRecords: function() {
+                    return this.record_list.length || 0;
+                }
+            },
+
+            methods: {
+                recordCountFormat: function(number) {
+                    return new Intl.NumberFormat().format(number);
+                },
+
+                recordType: function(type) {
+                    if (type === 'Collection') {
+                        return 'fa-archive';
+                    } else if (type === 'Folder') {
+                        return 'fa-folder';
+                    } else {
+                        return 'fa-file';
+                    }
+                },
+
+                numberOfColumns: function() {
+                    let screen_size = window.innerWidth;
```

Comment:
These break points will only take affect when the page first reloads, versus a resize, I assume?

Reply:
No, it should work on resize, too.

Reply:
Guess it's hard to tell right now since there are only 2 collections within an admin unit

Reply:
You can see the class reset if you have the console open.